.github/actions/zombienet-sdk/action.yml

```diff
@@ -58,6 +58,8 @@ runs:
       env:
         # zombienet-sdk expects RUN_IN_CI to be set when running in CI
         RUN_IN_CI: 1
+        # don't retry sdk tests
+        NEXTEST_RETRIES: 0
```

Comment:
We retry 6 times by default? Would setting this value to 0 increase our chances of failure?

Reply:
We set to 0 since we want to orchestrate the way we retry (since we are using spot instances and the failure could be from a claim of the underline vm).